atlasdb-persistent-lock-api/src/main/java/com/palantir/atlasdb/persistentlock/PersistentLockService.java

```diff
@@ -34,24 +32,22 @@
      * Attempt to acquire the lock.
      * Call this method before performing any destructive operations.
      * @param reason the reason for the lock, for logging purposes (e.g. "sweep")
-     * @return a {@link LockEntry} on success. The LockEntry will contain the given reason, and a unique ID. It is
-     *   essential that you retain a reference to this lock, as you will need it in order to release the lock.
+     * @return a {@link PersistentLockId} on success that represents the unique id of the lock that has
+     *   been acquired. You will need to provide this lock id when releasing the lock.
      */
     @POST // This has to be POST because we can't allow caching.
     @Path("acquire-backup-lock")
     @Produces(MediaType.APPLICATION_JSON)
-    LockEntry acquireBackupLock(@QueryParam("reason") String reason);
+    PersistentLockId acquireBackupLock(@QueryParam("reason") String reason);
```

Comment:
I don't think it should be a path param because reason can be something like "Needed for Rescue" or something like that.